code/modules/modular_computers/computers/subtypes/preset_wristbound.dm

```diff
@@ -18,6 +21,8 @@
 	card_slot = new /obj/item/computer_hardware/card_slot(src)
 	battery_module = new /obj/item/computer_hardware/battery_module(src)
 	battery_module.charge_to_full()
+	if(command)
+		tesla_link = new /obj/item/computer_hardware/tesla_link/small(src)
```

Comment:
why aren't we just overriding Initiialize() on the command subtype and handling this there? that's a better way of doing it than adding a variable